payone-adapter/src/main/java/com/commercetools/sunrise/payment/payone/PayonePaymentServiceProviderImpl.java

```diff
@@ -0,0 +1,51 @@
+package com.commercetools.sunrise.payment.payone;
+
+import com.commercetools.sunrise.payment.domain.CreatePaymentDataProvider;
+import com.commercetools.sunrise.payment.domain.PaymentServiceProvider;
+import com.commercetools.sunrise.payment.utils.PaymentPropertiesLoadingHelper;
+import io.sphere.sdk.payments.Payment;
+import io.sphere.sdk.payments.PaymentMethodInfo;
+import io.sphere.sdk.payments.PaymentStatus;
+
+import java.util.List;
+import java.util.Map;
+import java.util.function.BiFunction;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+/**
+ * Created by mgatz on 7/18/16.
+ */
+public class PayonePaymentServiceProviderImpl implements PaymentServiceProvider {
+
+    private PaymentPropertiesLoadingHelper propertiesLoadingHelper;
+
+    public PayonePaymentServiceProviderImpl() {
+        propertiesLoadingHelper = PaymentPropertiesLoadingHelper.createFromResource("methods/payone.properties");
```

Comment:
I would rather pass the configuration as parameters instead of deciding here how the configuration should be provided. I think this should be a decision of the developer (e.g. in Sunrise app I would like to put them integrated as Play Config). It also makes it easier to test.
If there are more than a pair of parameters, a class `PayoneConfiguration` with the required info is very useful, to both reduce the number of parameters and to extend them in the futrure.